Modules/posixmodule.c

```diff
@@ -11897,6 +11901,29 @@ os_urandom_impl(PyObject *module, Py_ssize_t size)
     return bytes;
 }
 
+#ifdef HAVE_MEMFD_CREATE_SYSCALL
+/*[clinic input]
+os.memfd_create
+
+    name: FSConverter
+    flags: unsigned_int(bitwise=True)
```

Comment:
According to PEP 446, all file descriptors should be non-inheritable by default. The default flags should be MFD_CLOEXEC.

Reply:
With the current implementation, both *name* and *flags* must be passed, so adding a default here will not have any effect. I'm not sure if *flags* should have a default value, but I will make that change if it is desired.